tests/unit/htdocs/modules/profile/ProfileVisibilityCsrfTest.php

```
@@ -0,0 +1,257 @@
+        $this->assertSame('POST', \Xmf\Request::getMethod());
+
+        $_SERVER['REQUEST_METHOD'] = 'GET';
+        $this->assertSame('GET', \Xmf\Request::getMethod());
+
+        // Restore
+        if ($origMethod !== null) {
+            $_SERVER['REQUEST_METHOD'] = $origMethod;
+        } else {
+            unset($_SERVER['REQUEST_METHOD']);
+        }
+    }
```

PHPMD (2.15.0)

[error] requestGetMethodReturnsCurrentMethod accesses the super-global variable $_SERVER. (undefined)

(Superglobals)